src/spryker/helm/app/templates/application/nginx/ingress-istio.yaml

```diff
@@ -2,21 +2,21 @@
 apiVersion: networking.istio.io/v1alpha3
 kind: VirtualService
 metadata:
- name: {{ .Release.Namespace }}-{{ .Values.prefix }}nginx-virtualservice
+ name: {{ .Release.Namespace }}-spryker-nginx-virtualservice
```

Comment:
Not sure this is necessary to hard code unlike others

Reply:
not intentional, will revert